src/agent/centralized_configuration/src/centralized_configuration.cpp

```diff
@@ -10,32 +13,49 @@ namespace centralized_configuration
     {
         try
         {
+            std::vector<std::string> groupIds {};
+            std::string messageOnSuccess {};
```

Comment:
Maybe we can use just a flag for this and create the message later?

Reply:
done!